src/main/java/core/basesyntax/strategy/DataConverterImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Action;
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    public List<FruitTransaction> convertToTransactions(List<String> dateList) {
+        List<FruitTransaction> result = new ArrayList<>();
+        for (String string : dateList) {
+            String[] stringArray = string.split(",");
```

Comment:
Consider using a constant for the comma delimiter to avoid magic strings and improve code readability.